python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ip-access-lists.j2

```diff
@@ -130,7 +126,14 @@ ip access-list {{ acl.name }}
 {%                     endif %}
 {%                 endif %}
 {%             endif %}
+{#             - only render if the string is not empty #}
```

Comment:
```suggestion
{#             Only render if the string is not empty #}
```

Reply:
It is not a style we usually have. Feel free to update elsewhere.

Reply:
this was done in last commit already